src/main/java/com/vistatec/ocelot/tm/gui/match/TranslationsPanel.java

```diff
@@ -0,0 +1,248 @@
+	private static final int DETACHED_COMP_WIDTH = 800;
+
+	/** The detached component height constant. */
+	private static final int DETACHED_COMP_HEIGHT = 400;
+
+	/** The table displaying matches. */
+	private TmTable matchesTable;
+
+	/** The table data model. */
+	private TranslationsMatchTableModel tableModel;
+
+	/** The scroll panel containing the table. */
+	private JScrollPane scrollPanel;
+
+	/**
+	 * The info panel displayed while loading matches or when the research
+	 * produces no results.
+	 */
+	private JPanel infoPanel;
+
+	/** The loading label. */
+	private JLabel lblLoading;
+
+	/** Label displayed when no matches are found. */
+	private JLabel lblNoResults;
+
+	/**
+	 * Constructor.
+	 * 
+	 * @param controller
+	 *            the controller.
+	 */
+	public TranslationsPanel(final TmGuiMatchController controller) {
+
+		super(controller);
+	}
+
+	/*
+	 * (non-Javadoc)
+	 * 
+	 * @see com.vistatec.ocelot.tm.gui.AbstractDetachableTmPanel#makePanel()
+	 */
+	@Override
+	protected void makePanel() {
+
+		// Configures the table.
+		configTable();
+		// Build the translation results label
+		JLabel lblTranslations = new JLabel("Translation Results");
+		lblTranslations.setFont(lblTranslations.getFont().deriveFont(Font.BOLD,
+				12));
+		// Build the panel
+		panel = new JPanel();
+		panel.setName("Translations");
+		JPanel translationsPanel = (JPanel) panel;
+		translationsPanel.setLayout(new GridBagLayout());
+		GridBagConstraints gridBag = new GridBagConstraints();
+
+		// Add the pin button.
+		gridBag.gridx = 0;
+		gridBag.gridy = 0;
+		gridBag.anchor = GridBagConstraints.NORTHWEST;
+		gridBag.insets = new Insets(10, 5, 10, 0);
+		translationsPanel.add(getPinComponent(), gridBag);
+
+		// Add the translations label.
+		gridBag.gridx = 1;
+		gridBag.gridy = 0;
+		gridBag.anchor = GridBagConstraints.NORTHWEST;
+		gridBag.insets = new Insets(10, 0, 10, 0);
+		translationsPanel.add(lblTranslations, gridBag);
+		// Add the scroll panel containing the table
+		gridBag.gridx = 0;
+		gridBag.gridy = 1;
+		gridBag.gridwidth = 2;
+		gridBag.weighty = 1;
+		gridBag.weightx = 1;
+		gridBag.insets = new Insets(0, 0, 0, 0);
+		gridBag.fill = GridBagConstraints.BOTH;
+		gridBag.anchor = GridBagConstraints.NORTH;
+		scrollPanel = new JScrollPane();
+		translationsPanel.add(scrollPanel, gridBag);
+
+		// Configure info panel and its labels.
+		lblLoading = new JLabel("Loading...");
+		lblNoResults = new JLabel("No results");
+		infoPanel = new JPanel(new FlowLayout(FlowLayout.LEFT));
+
+	}
+
+	/**
+	 * Configures the table.
+	 */
+	private void configTable() {
+
+		// Create the data model.
+		tableModel = new TranslationsMatchTableModel(null);
+		// Create and configure the table.
+		matchesTable = new TmTable();
+		matchesTable.setModel(tableModel);
+		// Make the table react to the ALT+R keystroke.
+		matchesTable.getInputMap(JComponent.WHEN_IN_FOCUSED_WINDOW).put(
+				KeyStroke.getKeyStroke('R', KeyEvent.CTRL_MASK),
+				"replaceTarget");
```

Comment:
Three things:
1) Ctrl-R is already in use on Windows to open `Filter > Rules`.  (We could change that, though.)
2) On the Mac, we use the command key for other accelerators, so may want to be consider that here.
3) We may eventually want to have a menu item somewhere that does the equivalent thing.  (Maybe in the `Segment` menu, unless there was enough TM-specific functionality to split off a separate menu.)